marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/graphVerifyTooltipString.cy.tsx

```diff
@@ -0,0 +1,64 @@
+import dataModelDisplaySettingsModal from "../../support/components/explore/data-model-display-settings-modal";
+import explore from "../../support/pages/explore";
+import graphExplore from "../../support/pages/graphExplore";
+import loginPage from "../../support/pages/login";
+import {ExploreGraphNodes} from "../../support/types/explore-graph-nodes";
+
+describe("Verify  text on hover nodes", () => {
+  beforeEach(() => {
+    cy.clearAllSessionStorage();
+    cy.clearAllLocalStorage();
+    cy.log("**Deleting hubCentralConfig file in FINAL**");
+    cy.deleteFiles("FINAL", "/config/hubCentral.json");
+
+    cy.log("**Deleting hubCentralConfig file in STAGING**");
+    cy.deleteFiles("STAGING", "/config/hubCentral.json");
+  });
+
+  after(() => {
+    cy.log("**Deleting hubCentralConfig file in FINAL**");
+    cy.deleteFiles("FINAL", "/config/hubCentral.json");
+
+    cy.log("**Deleting hubCentralConfig file in STAGING**");
+    cy.deleteFiles("STAGING", "/config/hubCentral.json");
+  });
+
+  it("Check info tooltip using default and email label", () => {
+    cy.loginAsDeveloperV2().withRequest();
+    loginPage.navigateToMainPage();
+    cy.visit("/tiles/explore");
+    cy.waitForAsyncRequest();
+    cy.wait(8000);
+
+    // Check info tooltip using
+    graphExplore.focusNode(ExploreGraphNodes.CUSTOMER_102);
+    graphExplore.getPositionsOfNodes(ExploreGraphNodes.CUSTOMER_102).then((nodePositions: any) => {
+      let customerCoordinates: any = nodePositions[ExploreGraphNodes.CUSTOMER_102];
+      const canvas = graphExplore.getGraphVisCanvas();
+      //Click on node to open side panel
+      canvas.trigger("mouseover", customerCoordinates.x, customerCoordinates.y, {force: true});
+      canvas.click(customerCoordinates.x, customerCoordinates.y, {force: true});
+    });
+    cy.wait(500);
+    cy.get(".vis-tooltip").contains(ExploreGraphNodes.CUSTOMER_102);
```

Comment:
If it is an assertion, please use .should("have.text", ExploreGraphNodes.CUSTOMER_102); since this make it easier to understand that we are validation something, and also, we have a mor accurate validation. Instead of check if contains, we will be validating if it is exactly what expected.